conf/pacific/upgrades/upgrade_from4x_big_cluster.yml

```diff
@@ -32,8 +32,7 @@ globals:
        role:
          - osd
          - rgw
-           #         - nfs
-           #         nfs upgrade is blocked till 5.1
+         - nfs
```

Comment:
Wouldn't this break any tests run with 5.0 build? IMHO, we should create a test specific for 5.1 and greater